### ISSUE TITLE
Frontend: Default to "all" if one of the input fields is empty

### ISSUE THREAD

**Issue by mbodeantor** [Police-Data-Accessibility-Project/data-sources-app#92]
### Context

Currently blocked by: 
#82 
https://github.com/Police-Data-Accessibility-Project/design-system/issues/22

### Requirements

* If a user leaves one field blank, the value add to the url for that variable should default to 'all' (support for this on the backend is in review https://github.com/Police-Data-Accessibility-Project/data-sources-app/pull/91)
* If a user leaves both fields blank, help text should appear telling them they need to fill out at least one field

_n.b. When the [`QuickSearchForm` component](https://github.com/Police-Data-Accessibility-Project/design-system/issues/22) is finished, it will handle this. Hold on this work until a new `design-system` version is published with the new component.

### Tests

* Request should be successfully submitted and results should be returned with at most one blank field, currently does not return any results for a blank field

**Comment by kalenluciano** [comment]
@josh-chamberlain said that @joshuagraber already fixed this. This worked for me in the https://pdap.dev/ website, so I think this issue is resolved already!

**Comment by joshuagraber** [comment]
> maybe already done on dev
 @josh-chamberlain @kalenluciano 
`pdap.io/dev`, yes, `2.0.4`; `data-sources/dev`, I goofed and bumped on `main`, which is still on `2.0.1`. That will cause conflicts when we merge `dev` -> `main`. So maybe we want to revert that PR on `main` and apply it to `dev` and deal with the conflicts there instead?

Regardless, both will need to be bumped to `2.0.5` from `^2.0.4` (note removing the caret, for ensuring install of specific version).
